backend/graphql/resolvers/product.resolver.ts

```diff
@@ -14,7 +22,7 @@ export default class ProductResolver {
     this.productService = new ProductService()
   }
 
-  @Authorized()
+  @UseMiddleware(isAdmin)
```

Comment:
This works, but you could have an actual decorator names `@IsAdmin` or something like this.

TypeGraphQL allows you to create custom middlewares/ decorators.

Feel like being challenged? lol